Dockerfile

```diff
@@ -342,9 +340,7 @@ RUN pip install bleach && \
     pip install qtconsole && \
     pip install six && \
     pip install terminado && \
-    # Latest version (6.0) of tornado breaks Jupyter notebook:
-    # https://github.com/jupyter/notebook/issues/4439
-    pip install tornado==5.0.2 && \
+    pip install tornado && \
```

Comment:
We should still test this carefully, there are some proxied requests, though hopefully none of them are broken by tornado.

Reply:
Any requests in particular I should try?

I can run a notebook just fine using the latest version of tornado but want to make sure that they are not specific steps I should perform to trigger some of these requests.

Other users seem to be reporting it is also working for them now with tornado 6.0.3: https://github.com/jupyter/notebook/issues/4439#issuecomment-582185303

Reply:
Cool, as long as you're running a session fine I think we're safe. We can always rollback if we see more issues.

Thanks for testing!